spec/models/conference_spec.rb

```diff
@@ -1639,4 +1639,26 @@
       expect(free_ticket.price_cents).to eq(0)
     end
   end
+
+  describe 'after_update' do
+    let(:conference) { create(:conference) }
+    let(:first_scheduled_event) { create(:event_scheduled, program: conference.program, hour: conference.start_date + conference.start_hour.hours) }
+    let(:second_scheduled_event) { create(:event_scheduled, program: conference.program, hour: conference.start_date + conference.end_hour.hours + 1.hour) }
```

Comment:
I would also add a scheduled event during the conference, so that we can be testing that we are only deleting the events outside conference hours, and not the other ones.